torch/onnx/_internal/fx/passes/modularization.py

```diff
@@ -214,12 +215,16 @@ def __init__(
                 _FX_TRACER_NN_MODULE_STACK_META_TYPE, _DYNAMO_NN_MODULE_STACK_META_TYPE
             ]
         ],
+        is_exported_program: bool = False,
     ):
         self._module_stack = []
         if nn_module_stack_meta is None:
             return
         raw_meta = copy.copy(nn_module_stack_meta)
         for item in raw_meta.items():
+            if is_exported_program:
```

Comment:
not sure what this is about ..

Reply:
It's to skip the first nn module stack. In exported program, it also records another call stack into it. Added comment